stdlib/public/core/VarArgs.swift

```diff
@@ -429,4 +430,388 @@ final public class VaListBuilder {
 
 #endif
 
+#else
+
+import CoreFoundation
```

Comment:
We can't use CoreFoundation in the standard library, that creates dependency loops.